src/encoded/types/dataset.py

```diff
@@ -666,3 +666,69 @@ class ReferenceEpigenome(Series):
     })
     def superseded_by(self, request, superseded_by):
         return paths_filtered_by_status(request, superseded_by)
+
+
+@collection(
+    name='experiment-series',
+    unique_key='accession',
+    properties={
+        'title': "Experiment series",
+        'description': 'A series that groups two or more experiments.',
+    })
+class ExperimentSeries(Series):
+    item_type = 'experiment_series'
+    schema = load_schema('encoded:schemas/experiment_series.json')
+    name_key = 'accession'
+    embedded = [
+        'biosample_ontology',
+        'contributing_awards',
+        'contributors',
+        'organism',
+        'related_datasets.lab',
+        'related_datasets.replicates.library.biosample',
```

Comment:
Do we need this and biosample_ontology?  Probably but I am not sure

Reply:
This is just for the following two in columns. I don't see they are very important.
```
        "related_datasets.replicates.library.biosample.age_display": {
            "title": "Biosample age",
            "type": "string"
        },
        "related_datasets.replicates.library.biosample.life_stage": {
            "title": "Life stage",
            "type": "array"
        },
```

Reply:
no we need those for batch_download, I forgot.